timeless/restaurants/models.py

```diff
@@ -4,17 +4,19 @@
 from timeless.db import DB
 from timeless.models import TimestampsMixin
 from timeless.poster.models import PosterSyncMixin
+from sqlalchemy_utils import ChoiceType
 
 
-class ReservationStatus(enum.Enum):
-    """Reservation status"""
-    unconfirmed = 1
-    confirmed = 2
-    started = 3
-    finished = 4
-    canceled = 5
-    late = 6
-    not_contacting = 7
+"""Reservation status"""
+RESERVATION_STATUS = [
+    (u"unconfirmed", u"Unconfirmed"),
```

Comment:
@emilianodellacasa `u"` prefix is used for `unicode` in python 2. in python 3 string type is unicode by default, so it's rudiment, just `("unconfirmed", "Unconfirmed")` will work the same way :)